src/Razor/test/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <DisablePubternalApiCheck>true</DisablePubternalApiCheck>
```

Comment:
Make this conditional on `$(TargetFramework)` too please

Reply:
In tests?

Reply:
Actually, this shouldn't be necessary at all because the `$(DisablePubternalApiCheck)` property is only read in the root Directory.Build.props and that check also excludes test projects.

Reply:
Same here.